client/src/components/DisplayTable/DisplayTable.tsx

```diff
@@ -509,6 +522,12 @@ export class _DisplayTable extends React.Component<
           util_components_with_defaults && "display-table-container--with-utils"
         )}
       >
+        <div className={"display-table-utils"}>
+          <div>{selectPageUtil}</div>
```

Comment:
Minor but this `div` doesn't seem needed

Reply:
I ended up keeping it so I could put the skip-to-data button before `selectPageUtil`